package.json

```diff
@@ -5,8 +5,11 @@
   "proxy": "http://localhost:5001/",
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.7.2",
-    "@patternfly/patternfly": "4.16.7",
-    "@patternfly/react-core": "4.23.1",
+    "@patternfly/patternfly": "4.23.3",
+    "@patternfly/react-core": "4.32.1",
```

Comment:
solution explorer usually only imports react-core and patternfly - did you add functionality that required icons and styles?